src/wallet/wallet.cpp

```diff
@@ -1040,6 +1044,38 @@ CAmount CWallet::GetChange(const CTransaction& tx) const
     return nChange;
 }
 
+
+
+// MVHF-BU Auto wallet backup
+bool CWallet::BackupWalletAuto(const std::string& strDest, int BackupBlock)
+{
+
+    boost::filesystem::path pathBackupWallet = strDest;
+
+    // Test for directory only, if so add default filename
+	if (boost::filesystem::is_directory(strDest))
```

Comment:
check with MVHF-BU-DES-WABU-2 and https://github.com/BTCfork/hardfork_prototype_1_mvf-bu/blob/master/doc/mvf-bu-test-design.md#411-sw-req-10-1

some more handling is needed here e.g. if it's not a path, but only a filename

also, please check indentation in this newly added method, it looks a little inconsistent